src/helpers.js

```diff
@@ -87,18 +93,34 @@ export function applySugar(environment: Object) {
 }
 
 export function getCommand(): { command: string, options: Object, parameters: Array<string> } {
+  const pythonFile = Path.join(Path.dirname(__dirname), 'index.py')
```

Comment:
Just run something like ``python -c 'import os;print("\0".join(map("=".join, dict(os.environ).items())))'``, no need to create a file.